### DISCUSSION TITLE
Got "You’re up to date!" when there's new version in appcast.xml

### DISCUSSION THREAD

**Discussion by rjyo** [sparkle-project/Sparkle#2626] [Q&A] [answered]
### Description of the problem
I can confirm my app has got the latest appcast.xml. But it just reports "0.2.1 is currently the newest version available."
Here's the link of it: https://download.usemurmur.com/releases/appcast.xml

I have these in my Info.plist

```
<key>CFBundleShortVersionString</key>
<string>0.2.1</string>
<key>CFBundleVersion</key>
<string>1020</string>
```

Here's the appcast.xml
```xml
...
<item>
<title>0.2.2</title>
<pubDate>Sun, 15 Sep 2024 02:03:15 +0900</pubDate>
<sparkle:channel>stable</sparkle:channel>
<sparkle:version>1030</sparkle:version>
<sparkle:shortVersionString>0.2.2</sparkle:shortVersionString>
<sparkle:minimumSystemVersion>14.0</sparkle:minimumSystemVersion>
<enclosure url="https://download.usemurmur.com/releases/Murmur-0.2.2.dmg" length="2574959" type="application/octet-stream" sparkle:edSignature="fxjmchkGKbsTZYmYyVT11s/GL2v06hTKn9g75B8vwLKFjDSQCRMqid3VhwUinhEk03ORZOdpj1uyII1oNTU9AQ=="/>
</item>
...
```

### Do you use Sandboxing in your app?
No

### Version of `Sparkle.framework` in the latest version of your app
2.6.4

### Version of `Sparkle.framework` in the old version of app that your users have (or N/A)
N/A

### Sparkle's output from Console.app
```
error	02:57:25.582154+0900	Murmur	Warning: Background app automatically schedules for update checks but does not implement gentle reminders. As a result, users may not take notice to update alerts that show up in the background. Please visit https://sparkle-project.org/documentation/gentle-reminders for more information. This warning will only be logged once.
```

### Steps to reproduce the behavior

[The more information provided and pasted verbatim, the easier it will be to diagnose an issue. If you can provide the affected application/binary and XML feed to reproduce an issue, share them]

**Comment by zorgiepoo** [comment, marked as answer]
You put the release on a channel named "stable", so your updater would have to have that set in the [allowed channels](https://sparkle-project.org/documentation/publishing/#channels).

Channels are usually used for beta updates. Typically you don't use a channel for stable releases since they are usually intended to be delivered to all users.

**Comment by rjyo** [comment]
Thank you so much for the swift answer! After adding a delegate for allowed channels, it worked like a charm.

Cursor added `--channel stable` to my build script and I thought it is the default channel... 

You saved my day! (actually 4am)

**Reply by zorgiepoo** [reply]
The default is no channel (or not passing --channel)

It makes little sense to use a channel if you intend for all your users to subscribe to that channel (that is typically the case for stable releases).
